jwql/utils/engineering_database.py

```diff
@@ -0,0 +1,174 @@
+#! /usr/bin/env python
+"""Module to interface the JWST Engineering Database.
+
+This module provides ``jwql`` with functions to interface and query the
+ JWST DMS Engineering Database.
+
+Authors
+-------
+
+    - Johannes Sahlmann
+
+Use
+---
+
+    This module can be imported and used with
+
+    ::
+
+        from jwql.engineering_database import query_single_mnemonic
```

Comment:
@Johannes-Sahlmann I think you mean `from jwql.utils.engineering_database` here?

Reply:
Corrected.